ACE/ace/OS_NS_fcntl.cpp

```diff
@@ -106,14 +106,25 @@ ACE_OS::open (const char *filename,
     ACE_FAIL_RETURN (h);
   else
     return h;
-#elif defined (INTEGRITY)
+#elif defined (ACE_LACKS_OPEN)
+  ACE_UNUSED_ARG (filename);
+  ACE_UNUSED_ARG (mode);
+  ACE_UNUSED_ARG (perms);
+  ACE_UNUSED_ARG (sa);
+  ACE_NOTSUP_RETURN (ACE_INVALID_HANDLE);
+#elif defined (ACE_INTEGRITY)
+# if defined (ACE_INTEGRITY178B)
```

Comment:
Can't the elif just check for `ACE_INTEGRITY178B`?

Reply:
ACE_LACKS_OPEN is added to the config file for integrity, just check this part again

Reply:
Yes, `ACE_LACKS_OPEN` is added because there is INTEGRITY-178 config where `open` is not available.

Reply:
Ok, I see that on line 109, but on line 115/116 you again check for integrity, where the code on line 117/118 is the same as on 133/134